doc/architecture/frontend/README.md

```diff
@@ -0,0 +1,209 @@
+# Architecture
+
+## 1. Component
+
+In our projects we follow this architecture:
+![Architecture](assets/architecture.png)
+
+### 1.1 Smart group
+
+Logic and data processing. Usual supported by developers.
+
+#### 1.1.1 Container
+
+Main question:\
+What does business data component show and business logic
+component use? 
+
+Responsible for:
+1. Getting and preparing data from store
+2. Preparing actions
+3. Including business logic
+
+Filename:\
+`<component>.container.tsx`
+
+Exports `mapStateToProps` and / or `mapDispatchToProps` functions.
```

Comment:
Do we want to export functions? Maybe should we export connected component? We export functions to be able to test them, right? However if we do not export connected component, there is something wrong in naming of this file because container is a connected component.